packages/react/src/dropdown-menu/DropdownMenuPanel.tsx

```diff
@@ -41,6 +41,9 @@ export function DropdownMenuPanel({
 	const { onKeyDown } = useKeydownNavigation();
 
 	const { style, ref: popoverRef } = popover.getPopoverProps();
+	const { matchReferenceWidth } = popover.getOptions();
+
+	if (!isMenuOpen) return null;
```

Comment:
This was updated to improve anchoring performance, as recommended here https://floating-ui.com/docs/react#react

> Ensure you are using conditional rendering for the floating element, not an opacity/visibility/display style